snmp/datadog_checks/snmp/data/default_profiles/zebra-printer.yaml

```diff
@@ -0,0 +1,92 @@
+extends:
+  - _base.yaml
+
+metadata:
+  device:
+    fields:
+      vendor:
+        value: "zebra"
+      version:
+        symbol:
+          MIB: ZEBRA-MIB
+          OID: 1.3.6.1.4.1.10642.1.18.0
+          name: zbrGeneralLINKOSVersion
+      model:
+        symbol:
+          MIB: ZEBRA-MIB
+          OID: 1.3.6.1.4.1.10642.1.1.0
+          name: zbrGeneralModel
+      os_version:
+        symbol:
+          MIB: ZEBRA-MIB
+          OID: 1.3.6.1.4.1.10642.1.2.0
+          name: zbrGeneralFirmwareVersion
```

Comment:
Those were `metric_tags` initially.

Reply:
Should `zbrGeneralLINKOSVersion` mapped to `os_version` instead ?

And maybe `zbrGeneralFirmwareVersion` to `version` (version of the device) ?